### ISSUE TITLE
ObjectType and accept_definitions gone in 2.x?

### ISSUE THREAD

**Issue by digitalmoksha** [rmosolgo/graphql-ruby#4002]
I tried up using the 2.x gem, and getting an error on

```ruby
GraphQL::ObjectType.accepts_definitions(authorize: GraphQL::Define.assign_metadata_key(:authorize))
```

It looks like `GraphQL::ObjectType` has been removed? The only remnant I see are in method comments.

**Comment by rmosolgo** [comment]
Yes, that's correct -- the legacy definition system is completely removed in GraphQL-Ruby 2.0. Were you using it in 1.13.x without deprecations? If so, it means I missed a spot 🙈 !

**Comment by rmosolgo** [comment]
Ah... but did they _do_ anything? I mean, if you just delete them, does your application behavior change at all?

**Comment by digitalmoksha** [comment]
Honestly, don't know yet, so many other things are broken, I can't determine cause and effect yet 😃. I'll keep digging into it and post back

**Comment by tim-heighes-fishbrain** [comment]
@rmosolgo Can we expect an update to graphql-pro? Version 1.22.1 is triggering this deprecation warning.

**Comment by rmosolgo** [comment]
Oops -- thanks for pointing that out! I just released 1.22.2 which should silence those warnings. Please let me know if you have any more trouble with it.
